vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/scripts/new_accel_brake_map_server.py

```diff
@@ -39,11 +39,27 @@ def __init__(self):
             CalibData, "/accel_brake_map_calibrator/get_data_service", self.get_data_callback
         )
 
-        package_path = get_package_share_directory("accel_brake_map_calibrator")
         default_map_path = get_package_share_directory("raw_vehicle_cmd_converter")
+        self.declare_parameter(
+            "/accel_brake_map_calibrator/csv_default_map_dir", default_map_path + "/data/default/"
+        )
+        self.default_map_dir = (
+            self.get_parameter("/accel_brake_map_calibrator/csv_default_map_dir")
+            .get_parameter_value()
+            .string_value
+        )
+
+        package_path = get_package_share_directory("accel_brake_map_calibrator")
+        self.declare_parameter(
+            "/accel_brake_map_calibrator/csv_calibrated_map_dir", package_path + "/config/"
+        )
+        self.calibrated_map_dir = (
+            self.get_parameter("/accel_brake_map_calibrator/csv_calibrated_map_dir")
+            .get_parameter_value()
+            .string_value
+        )
 
-        self.default_map_dir = default_map_path + "/data/default/"
-        self.calibrated_map_dir = package_path + "/config/"
+        self.calibration_method = "each_cell"
```

Comment:
Can we use the other calibration method here? If so, I think it is better to use a parse argument.

Reply:
@shmpwk 
I fixed this commit. https://github.com/autowarefoundation/autoware.universe/pull/2509/commits/d16640432e595a6d0a543dfa4b9c0df7a72683f5

And I changed to get default parameter from ros parameter instead of argument.